TDD_RecentFileList/src/recentFileListPackage/_TestRecentFileList.java

```diff
@@ -0,0 +1,23 @@
+package recentFileListPackage;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class _TestRecentFileList {
+
```

Comment:
We need to have at-least one test per each public method , here I think we can use the @before, @beforeClass  , @after , @afterClass